api/src/main/java/com/cloud/management/ManagementServerHost.java

```diff
@@ -14,19 +14,23 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package com.cloud.cluster;
+package com.cloud.management;
```

Comment:
if we are renaming a package it should be as org.apache.cloudstack.<package>

Reply:
Ok, will do the change